src/sentry/reprocessing2.py

```diff
@@ -208,7 +218,7 @@ def reprocess_event(project_id, event_id, start_time):
         with sentry_sdk.start_span(op="reprocess_event.set_attachment_meta"):
             attachment_cache.set(cache_key, attachments=attachment_objects, timeout=CACHE_TIMEOUT)
 
-    preprocess_event_from_reprocessing(
+    preprocess_event_from_reprocessing.delay(
```

Comment:
Delaying preprocessing here sounds expensive, given that we already have the event payload loaded. You could be just passing the event payload in.